populate_course_run_reporting_type_and_announcement.py

```diff
@@ -0,0 +1,80 @@
+import argparse
+import datetime
+from dateutil.parser import parse
+import json
+import logging
+import os
+
+import logging
+
+from edx_sailthru.sailthru_content.services.catalog_api_service import CatalogApiService
```

Comment:
Another alternative here is to simply not share code with this module. Just use `EdXRestApiClient` directly. There is nothing special about what the Sailthru code is doing.